tools/sof-get-kernel-line.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+log_file=${1:-"/var/log/kern.log"}
+bootup_keyword='\[    0.000000\] Linux version '
+grep -na "$bootup_keyword" $log_file|awk -F ':' '{print $1 $(NF-2);}'|sed "s/$bootup_keyword//g"|awk '{print $1, $2;}'
```

Comment:
```suggestion
grep -na "$bootup_keyword" $log_file|awk -F ':' '{print $1 $(NF-2);}'|sed "s/$bootup_keyword//g"|awk '{print $1, $2;}' | tac
```
use a simple tac to output in reserved order

Reply:
here you try the small file on the terminal,  
if you file is large like our BDW:  1.8G  
your line number will be >1000 so here your terminal output will been flash